### ISSUE TITLE
Module not found: Error: Can't resolve 'he' ...

### ISSUE THREAD

**Issue by maiermuc** [vuejs-templates/webpack-simple#41]
after clean install and run
`npm run dev` 
or 
`npm run build `
following issue

```
ERROR in ./~/vue/dist/vue.js
Module not found: Error: Can't resolve 'he' in 'test\node_modules\vue\dist'
 @ ./~/vue/dist/vue.js 5762:11-24
 @ ./src/main.js
 @ multi main
```

The demo 'Welcome Site' is working.

**Comment by davewood** [comment]
same problem here, funny enough "he" is listed in the vuejs devDependencies

  "devDependencies": {
    //...
    "he": "^1.1.0",

**Comment by davewood** [comment]
`npm install he` as a temporary workaround to get rid of the error message.
I'm not sure if it should be added to 'dependencies' so I'm not sending a PR

**Comment by davewood** [comment]
note: without 'he' installed hot reload doesn't seem to work.

**Comment by maiermuc** [comment]
`npm install he` works for me

thanks

**Comment by maiermuc** [comment]
is solved with vue@2.0.7
